### PR TITLE
[T8] Add signUp UI page

### PR DESCRIPTION
**Pull request by Animansoubi**
add a logo
add custom button
add custom edit text
add icons

### CODE REVIEW

File: app/src/main/java/com/gathering/android/MainActivity.kt [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+import android.content.Intent
 import android.os.Bundle
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        findViewById<Button>(R.id.btn_test).setOnClickListener {
```
Comment (moyarsuccess) [inline app/src/main/java/com/gathering/android/MainActivity.kt:12]: You should use view binding here

File: app/src/main/java/com/gathering/android/SignUpActivity.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.gathering.android
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+class SignUpActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_sign_up)
```
Comment (moyarsuccess) [inline app/src/main/java/com/gathering/android/SignUpActivity.kt:9]: Same as above, use view binding please.

File: app/src/main/res/layout/activity_sign_up.xml [hunk 3]
```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/back_sign_up">
+
+    <com.google.android.material.textfield.TextInputLayout
```
Comment (moyarsuccess) [inline app/src/main/res/layout/activity_sign_up.xml:8]: Please use styles here to manage the duplicated resources and configurations that was used here.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,3 +1,7 @@
 <resources>
     <string name="app_name">Gathering</string>
+    <string name="sign_up">Sign up</string>
+    <string name="email">Email</string>
+    <string name="enter_password">Password</string>
+    <string name="enter_password_again">Please Enter your password again</string>
```
Comment (moyarsuccess) [inline app/src/main/res/values/strings.xml:6]: Change to Confirm password

**Review by moyarsuccess — CHANGES_REQUESTED** [review]

**Review by idaosk — APPROVED** [review]

**Pull request closed without merging**